adafruit_si5351.py

```diff
@@ -169,10 +169,11 @@ def _configure_registers(self, p1, p2, p3):
             self._si5351._write_u8(_SI5351_REGISTER_177_PLL_RESET, (1 << 7) | (1 << 5))
 
         def configure_integer(self, multiplier):
-            """Configure the PLL with a simple integer mulitplier for the most
+            """Configure the PLL with a simple integer multiplier for the most
             accurate (but more limited) PLL frequency generation.
             """
-            assert 14 < multiplier < 91
+            if multiplier > 91 or multiplier < 14:
```

Comment:
Might need to add `=` as well? For the edge cases of 14 and 91? Also, change message to state acceptable range, like "foo must be in range 14 to 91" (or whatever correct values are) instead of "for must be between 14 and 91" to help remove confusion about boundary values.

Reply:
I correct this one, and the other Exception in the file to include this suggestion. Thanks is in https://github.com/adafruit/Adafruit_CircuitPython_SI5351/pull/21/commits/ad530a39642b04810407cc6af042ace504c38c83